docs/reference/commandline/search.md

```diff
@@ -61,37 +63,59 @@ This example displays images with a name containing 'busybox':
     marclop/busybox-solr
 
-### Search images by name and number of stars (-s, --stars)
+### Display non-truncated description (--no-trunc)
+
+This example displays images with a name containing 'busybox',
+at least 3 stars and the description isn't truncated in the output:
+
+    $ docker search --stars=3 --no-trunc busybox
+    NAME                 DESCRIPTION                                                                               STARS     OFFICIAL   AUTOMATED
+    busybox              Busybox base image.                                                                       325       [OK]       
+    progrium/busybox                                                                                               50                   [OK]
+    radial/busyboxplus   Full-chain, Internet enabled, busybox made from scratch. Comes in git and cURL flavors.   8                    [OK]
+
+## Filtering
+
+The filtering flag (`-f` or `--filter`) format is a `key=value` pair. If there is more
+than one filter, then pass multiple flags (e.g. `--filter "foo=bar" --filter "bif=baz"`)
+
+The currently supported filters are:
+
+* stars (int - number of stars the image has)
+* is-automated (true|false) - is the image automated or not
+* is-official (true|false) - is the image official or not
+
+
+### stars
 
 This example displays images with a name containing 'busybox' and at
 least 3 stars:
 
-    $ docker search --stars=3 busybox
+    $ docker search --filter stars=3 busybox
     NAME                 DESCRIPTION                                     STARS     OFFICIAL   AUTOMATED
     busybox              Busybox base image.                             325       [OK]       
     progrium/busybox                                                     50                   [OK]
     radial/busyboxplus   Full-chain, Internet enabled, busybox made...   8                    [OK]
 
 
-### Search automated images (--automated)
+### is-automated
 
-This example displays images with a name containing 'busybox', at
-least 3 stars and are automated builds:
+This example displays images with a name containing 'busybox'
+and are automated builds:
 
-    $ docker search --stars=3 --automated busybox
+    $ docker search --filter is-automated busybox
     NAME                 DESCRIPTION                                     STARS     OFFICIAL   AUTOMATED
     progrium/busybox                                                     50                   [OK]
     radial/busyboxplus   Full-chain, Internet enabled, busybox made...   8                    [OK]
 
+### is-official
 
-### Display non-truncated description (--no-trunc)
+This example displays images with a name containing 'busybox', at least
+3 stars and are official builds:
 
-This example displays images with a name containing 'busybox',
-at least 3 stars and the description isn't truncated in the output:
+    $ docker search --filter "is-automated=true stars=3" busybox
```

Comment:
good catch indeed. Even though this may work, I don't think we ever documented it like this, perhaps we should change it to multiple `--filter` args in the example? Or do we want to support this notation?